deploy/charts/cert-manager/values.yaml

```diff
@@ -317,6 +317,17 @@ podLabels: {}
 #     - "1.1.1.1"
 #     - "8.8.8.8"
 
+# Optional hostAliases, useful if you have issues with the selfCheck.
```

Comment:
suggestion: Could we expand on this comment a little more and make clear it only applies to the controller?

```suggestion
# Optional hostAliases for cert-manager-controller pods. May be useful when performing ACME DNS-01 self checks.
```